proto/lbm/token/v1/tx.proto

```diff
@@ -16,39 +16,22 @@ service Msg {
   // - EventSent
```

Comment:
I knew the `EventSent` event isn't existed in daphne version. Is't it? 
But still it is remained. why?

Reply:
Our clients require to add the new typed events to the old APIs.